docs/snippets/common/component-story-custom-source.js.mdx

```diff
@@ -1,13 +1,19 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const CustomSource = () => Template.bind({});
+import Button from '.Button';
```

Comment:
```suggestion
import Button from './Button';
```

Reply:
Thanks for that, really appreciate it. Gonna commit the change and mark this as solved.